contracts/ControllerV1.sol

```diff
@@ -89,7 +94,7 @@ contract ControllerV1 is IControllerV1, SafeTeller, Ownable {
         uint256 expectedPodId,
         string memory _imageUrl
     ) external override {
-        address safe = createSafe(_members, threshold);
+        address safe = createSafe(_members, threshold, expectedPodId);
```

Comment:
are we verifying expected pod id somewhere?

Reply:
The `_createPod` function this ultimately calls checks to see if that value aligns with whatever the next expected pod ID should be. The `recoverSafe` function doesn't verify anywhere, but that's kind of by design right?